e2e2/test/cases/nvidia/manifests/efa-device-plugin.yaml

```diff
@@ -90,7 +155,16 @@ spec:
           volumeMounts:
             - name: device-plugin
               mountPath: /var/lib/kubelet/device-plugins
+            - name: infiniband-volume
+              mountPath: /dev/infiniband
+          resources:
+            requests:
+              cpu:    10m
+              memory: 20Mi
```

Comment:
what is this for?

Reply:
It is synced from https://github.com/aws-samples/aws-efa-eks/blob/main/manifest/efa-k8s-device-plugin.yml

Reply:
we should be using this https://github.com/aws/eks-charts/tree/master/stable/aws-efa-k8s-device-plugin